docs/components/Code.tsx

```diff
@@ -53,9 +55,11 @@ const PlaceholderImage = () => (
 function LiveCode({
 	showCode = false,
 	enableProse = false,
+	headingType,
 }: {
 	showCode?: boolean;
 	enableProse?: boolean;
+	headingType?: 'h2' | 'h3' | 'h4';
```

Comment:
This prop name is a bit weird... should it be something like `exampleHeadingType`,  `exampleContentHeadingType` or something like that?